### PR TITLE
Test project setup

### PR DESCRIPTION
**Pull request by nschenone**

### CODE REVIEW

File: test_project_setup/project.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+kind: project
+metadata:
+  name: test-project-setup
+spec:
+  params:
+    source: https://github.com/mlrun/test-notebooks#main
+    secrets_file: mock_secrets.env
+    default_image: mlrun/mlrun:1.5.1
```
Comment (MichaelSkralivetsky) [inline test_project_setup/project.yaml:8]: remove hardcoded 1.5.1

File: test_project_setup/project.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    handler: train_model
+  workflows:
+  - path: src/main_workflow.py
+    name: main
+  artifacts: []
+  conda: ''
+  source: https://github.com/mlrun/test-notebooks#main
+  load_source_on_run: true
+  desired_state: online
+  default_image: mlrun/mlrun:1.5.1
```
Comment (MichaelSkralivetsky) [inline test_project_setup/project.yaml:26]: remove hardcoded 1.5.1

**Review by MichaelSkralivetsky — CHANGES_REQUESTED** [review]

File: test_project_setup/test_project_setup.ipynb [hunk 3]
```diff
@@ -0,0 +1,517 @@
+{
```
[the remark this answers is not in the document]
Comment (nschenone) [reply test_project_setup/test_project_setup.ipynb:1]: <p>Not sure I understand. Random context?</p>

[the remark this answers is not in the document]
Comment (MichaelSkralivetsky) [reply test_project_setup/test_project_setup.ipynb:1]: context="./some_directory"

**Pull request merged**